src/runtime/transition/index.ts

```diff
@@ -178,7 +178,10 @@ export function draw(node: SVGElement & { getTotalLength(): number }, {
 	duration,
 	easing = cubicInOut
 }: DrawParams = {}): TransitionConfig {
-	const len = node.getTotalLength();
+	let len = node.getTotalLength();
+	if (getComputedStyle(node).strokeLinecap !== 'butt') {
+		len += parseInt(getComputedStyle(node).strokeWidth);
```

Comment:
This was merged, so I'm a little late possibly, but what are the performance implications of calling `getComputedStyle` twice here? It at least could be deduplicated to one call.

Reply:
Oh, whoops. It's probably not ideal. I'd definitely merge a PR that just called it once and saved the value.